assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -0,0 +1,33 @@
+[id="assembly-rhdh-telemetry_{context}"]
+= Telemetry data collection
+
+The telemetry data collection feature is enabled by default. {product} sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin.
```

Comment:
```suggestion
The telemetry data collection feature is disabled by default. If enabled, {product} will send telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin.
```

Reply:
Per the RH supplementary style guide (and IIRC, maybe the IBM one too) we actually discourage using the future tense unless absolutely necessary, so I'd leave this as is, or "If the feature is enabled, {product} sends telemetry data...". See https://redhat-documentation.github.io/supplementary-style-guide/#will

Reply:
:as-you-wish: